guides/source/action_text_overview.md

```diff
@@ -8,149 +8,145 @@ rich text content.
 
-Instances of `ActionText::RichText` are capable of safely rendering themselves, so embed them directly into the page:
+Instances of `ActionText::RichText` can be directly embedded into a page because they have already sanitized their content for a safe render. You can display the content as follows:
 
 ```erb
-<%= @message.content %>
+<%= @blog.content %>
 ```
 
-Learn more about Action Text's sanitization process in the documentation for the [`ActionText::RichText`](https://api.rubyonrails.org/classes/ActionText/RichText.html) class.
+`ActionText::RichText#to_s` safely transforms RichText into an HTML String. On the other hand `ActionText::RichText#to_plain_text` returns a string that is not HTML safe and should not be rendered in browsers. You can learn more about Action Text's sanitization process in the [documentation](https://api.rubyonrails.org/classes/ActionText/RichText.html) for the `ActionText::RichText` class.
 
-NOTE: If there's an attached resource within `content` field, it might not show properly unless you
-have *libvips/libvips42* package installed locally on your machine.
-Check their [install docs](https://www.libvips.org/install.html) on how to get it.
+NOTE: If there's an attached resource within `content` field, it might not show properly unless you have the necessary [dependencies for Active Storage](active_storage_overview.html#requirements) installed.
 
-By default, Action Text will render rich text content inside an element with the `.trix-content` class. Elements with this class, as well as the Action Text editor, are styled by the [`trix` stylesheet](https://unpkg.com/trix/dist/trix.css). To provide your own styles instead, remove `trix` from the pre-processor directives in the `app/assets/stylesheets/actiontext.css` file created by the installer:
+## Customizing the Rich Text Content Editor (Trix)
 
-```diff
- /*
-  * Provides a drop-in pointer for the default Trix stylesheet that will format the toolbar and
-  * the trix-editor content (whether displayed or under editing). Feel free to incorporate this
-  * inclusion directly in any other asset bundle and remove this file.
-  *
-- *= require trix
- */
+There may be times when you want to update the presentation of the editor to meet your stylistic requirements, this section guides on how to do that.
+
+### Removing or Adding Trix Styles
+
+By default, Action Text will render rich text content inside an element with the `.trix-content` class. This is set in `app/views/layouts/action_text/contents/_content.html.erb`.  Elements with this class are then styled by the trix stylesheet.
+
+If you’d like to update any of the trix styles, you can add your custom styles in `app/assets/stylesheets/actiontext.css`.
+
+However, if you’d prefer to provide your styles instead of using the trix stylesheet, you can remove trix from the pre-processor directives in the `app/assets/stylesheets/actiontext.css` file by deleting the following:
+
+```css
+= require trix
```

Comment:
This will be merged for Rails 8, right? in that case, I think this should be `@import url('trix.css');` since Propshaft will become the default asset pipeline instead of Sprockets

Reply:
We might have to update it when the default changes, but for now I believe this is how things are generated still, so we should keep that way for current compatibility.